CONTRIBUTING.md

```diff
@@ -66,15 +69,38 @@ TIP: if you want you can use prefix eg "CSS" or "JS" for branch naming and in yo
 - make sure to use only class and id
 - Try to use ids or combination of selectors so we won't mix our styles
 - If you decide to use classes use prefix of your task which you are working on eg  `footer-link` `nav-search-button`
-- Avoid tag name like main or section or h1 or p this can mix your styles with other styling or you can also use inline styling in your elements
+- Avoid tag name like `main`, `section` , `h1` , `p` , `header` and `footer` for styling this can mix your styles with other styling or you can use combination of selectors `.parent p + p` or `.parent p ~ p`
+- Avoid Inline Styling
 
 ### HTML
 
+- make sure to install and use [W3C Web Validator](https://marketplace.visualstudio.com/items?itemName=CelianRiboulet.webvalidator) VSCode Extension help you to validate your Html file
 - use Semantic html like  `section` `article` tags for content
 - avoid generic html and use `div` and `span` only for styling purposes
 
 ### Assets
 
 - Assets includes images that used on pages.
+- Always make sure to use [TINYpng](https://tinypng.com/) to compress the size of your images before you push it
+
+### Eslint
+

```

Comment:
Add how to use eslint? Do we need to add some specific code on the top of our JavaScript file content
before our code for the linter to work?

Reply:
you can run `npm lint` or `npx eslint js/app/app.js` in the terminal you will see errors if there is any

Reply:
I understand. So
 "env": {
    "browser": true,
    "node": true
}
used instead of using 
/* eslint-env node, browser */
in the top of a file.